src/Modules/StatCan.OrchardCore.Assessment/Views/Widget-Assessment.liquid

```diff
@@ -0,0 +1,46 @@
+<div id="surveyElement" style="display:inline-block;width:100%;"></div>
+<div id="surveyResult"></div>
+
+{% style name:"default", src:"https://unpkg.com/survey-knockout@1.8.24/modern.css" %}
+{% script name:"knockout", src:"https://cdnjs.cloudflare.com/ajax/libs/knockout/3.5.1/knockout-latest.js", at:"Foot" %}
+{% script name:"survey-ko", src:"https://unpkg.com/survey-knockout@1.8.24/survey.ko.min.js", at:"Foot" %}
+
+{% block "script", at: "Foot", depends-on:"survey-ko" %}
+
+Survey
+    .StylesManager
+    .applyTheme("default");
+
+Survey
+    .JsonObject
+    .metaData
+    .addProperty("itemvalue", {name: "score:number"});
+
+var json = {{Model.ContentItem.Content.Assessment.Data.Text | raw}};
```

Comment:
This is an unsafe way to pass json to the client. You should put this json in a data-attribute of the html dom element used to render the SurveyJs component. Then, parse it in your script.